README.md

```diff
@@ -67,6 +67,199 @@ Or install the plugin manually:
     - Windows: `C:\Users\USER\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins`
     - Mac OS: `Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins`
 
+## Development Setup
+
+### Requirements:
+- QGIS-LTR (3.16)
```

Comment:
Does this also work with older QGIS versions? Given the tools minimal requirement of 3.4 - it should?

Reply:
This is the QGIS installation used during development, which should be at least the LTR version.
Compatibility should go down to the minimal requirement, but development should be done on LTR and in the best case also have upcoming changes in mind (3.22 available already). wdyt?

Reply:
ah yes, we're talking about the dev setup here. True, I agree.